test/components/admin/editions/editorial_remark_component_test.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class Admin::Editions::EditorialRemarkComponentTest < ViewComponent::TestCase
+  test "it constructs output based on the editorial remark when an author is present" do
+    editorial_remark = build_stubbed(:editorial_remark, created_at: Time.zone.local(2020, 1, 1, 11, 11))
+    author = editorial_remark.author
+    render_inline(Admin::Editions::EditorialRemarkComponent.new(editorial_remark:))
+
+    assert_equal page.text, "\n  #{editorial_remark.body}\n  #{author.name}  1 January 2020 11:11am\n\n"
```

Comment:
**Opinion:** you'd be able to make this a bit less pernickety by stripping whitespace:

```suggestion
    assert_equal page.text.strip, "#{editorial_remark.body}\n  #{author.name}  1 January 2020 11:11am"
```

Although it doesn't solve the internal whitespace within the string. So may not be all that useful 🤷🏻

Reply:
Thanks. I still think it's better. I'll update it now